pkg/ssh/ssh_test.go

```diff
@@ -48,6 +48,26 @@ func TestExec(t *testing.T) {
 	require.Error(t, err, "failed to connect: ssh: handshake failed: ssh: disconnect, reason 2: Too many authentication failures")
 }
 
+func TestExecWithInput(t *testing.T) {
+	options := ConnectionExecOptions{
+		Port: 22,
+		Host: "localhost",
+		Args: []string{"md5sum"},
+	}
+
+	input, err := os.Open("/etc/fstab")
+	require.Error(t, err, nil)
```

Comment:
By looking at other code in this repo, I get the sense that perhaps this should be
```go
    require.NoError(t, err)
            ^^
```
? (or `NoErrorf-with-a-message`?)

Reply:
Testing on fedora-39 succeeds now.